---
title: "Lon and GroEL Trees"
output: html_document
---

```{r setup, echo = FALSE, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)

library(ape)
library(phytools)
library(ggtree)
```

```{r fig.width=5, fig.height=5}
#create a tree using the fasta treefile of aligned lon protease sequences
lon.phy = read.tree("lon.aligned.fasta.treefile")

#in order to have comparable phylogenies, drop tips that aren't in GroEL phylo
lon.phy.reduced = drop.tip( lon.phy, c("Meiothermus_taiwanensis_4YPL_A", "Bacillus_subtilis_3M6A_A", "Yersinia_pestis_6ON2_A", "Yersinia_pestis_6V11_A", "Escherichia_coli_6U5Z_A") )

#root tree with E. coli strain 6N2I A
rooted.lon = root(lon.phy.reduced, "Escherichia_coli_6N2I_A", resolve.root = TRUE)

#make ultrametric
lon.ultra = force.ultrametric(rooted.lon)

plot(lon.ultra)
edgelabels(lon.ultra$edge.length)

```

```{r fig.width=7, fig.height=5}

#create a tree using the fasta treefile of aligned groel sequences
groel.phy = read.tree("groel.aligned.fasta.treefile")

#drop 1 e coli so that the other e coli can be an outgroup
groel.phy.reduced = drop.tip( groel.phy, c("Escherichia_coli_4PKN_A") )

#root the tree with E. coli strain K-12 5OPX A
rooted.groel = root(groel.phy.reduced, "Escherichia_Coli_K-12_5OPX_A")

#make ultrametric
groel.ultra = force.ultrametric(rooted.groel)

plot(groel.ultra)
edgelabels(groel.ultra$edge.length)
#edgelabels()
```

```{r}
lon.ultra$edge.length
groel.ultra$edge.length

is.rooted(rooted.lon)
is.rooted(rooted.groel)

```


```{r}
phytable = read.csv(file = "Lon.Groel.EdgeLengths.csv", header = TRUE, sep = ",")
phytable
#f test to see statistical difference between edge lengths and take variance of groups into account
var.test(phytable$EL..EL.of.outgroup ~ phytable$phylogeny)


#want more sensitive model, look in output files (next one post - GTR, etc. - see earlier exercises)
#justify it, no matter how accurate it is 
#every base pair can change (no clamping, allows high variablity)
#making model decisions: use some knowledge of biochemistry to make guesses 

```
```{r}

phytable = read.csv(file = "Lon.Groel.Ttest.csv", header = TRUE, sep = ",")
phytable
#t test to see statistical difference between edge lengths and take variance between groups
t.test(phytable$A, phytable$B)


```








